src/vehicle.cpp

```diff
@@ -2986,6 +2986,24 @@ bool vehicle::is_moving() const
     return velocity != 0;
 }
 
+bool vehicle::can_use_rails() const
+{
+    bool canUse = true;
```

Comment:
Could you please replace camelCase with underscores?

Reply:
Sure, other languages sometimes popup